ttnn/cpp/ttnn/operations/moreh/moreh_dot/device/moreh_dot_program_factory.cpp

```diff
@@ -16,30 +15,30 @@ MorehDotOperation::SingleCore::cached_program_t MorehDotOperation::SingleCore::c
     using namespace tt;
     using namespace tt::tt_metal;
 
-    const auto& input_tensor_a = tensor_args.input_tensor_a;
-    const auto& input_tensor_b = tensor_args.input_tensor_b;
-    auto& output_tensor = tensor_return_value;
+    const auto& input_a = tensor_args.input_a;
+    const auto& input_b = tensor_args.input_b;
+    auto& output = tensor_return_value;
```

Comment:
You did not resolve this. I commented above.

Reply:
Sorry, I reverted without knowing